packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -141,14 +164,15 @@ export function canBeInvited(invitee: UserWithMembership, team: TeamWithParent)
   return INVITE_STATUS.CAN_BE_INVITED;
 }
 
-export async function getExistingUsersToInvite({
-  usernamesOrEmails,
+export async function findUsersWithInviteStatus({
```

Comment:
get -> find. Informs that it isn't simply reading it. It is actually doing a lookup somewhere.